### ISSUE TITLE
Quick copy for action log?

### ISSUE THREAD

**Issue by yp05327** [go-gitea/gitea#26116]
### Feature Description

Maybe it is good to add a button here to copy the logs of this step?
![image](https://github.com/go-gitea/gitea/assets/18380374/ce5e9161-ff0f-4dbf-8bc8-4f2cd3de5d26)

And for each line in the step, you can quickly copy the log by double-click?

**Comment by lunny** [comment]
There is a download button?

**Comment by puni9869** [comment]
The download button will be a good option or showing the row format in next tab.

**Comment by lunny** [comment]
I mean we already have a log file download button, is this still necessary?

**Comment by sebthom** [comment]
The download button downloads the complete log of all steps. I also would find it quite useful to quickly get the plain log output of a single step as proposed here.

**Comment by sebthom** [comment]
@puni9869 that's how I understand this proposal and I find it quite useful

**Comment by puni9869** [comment]
Yeah, I also like this proposal,   just to addon, I have worked on many ci platforms, they also give raw format of logs open in different tab, shall we also add that too.

**Comment by yp05327** [comment]
> I mean we already have a log file download button, is this still necessary?

The downloaded log files are raw data from runner, and are not divided by steps, so it is hard to read and get the information what users want.
So I think users would be prefer to copy logs from web.

**Comment by puni9869** [comment]
Ok, lets limit this with copy option and we will circle back if required.

**Comment by sebthom** [comment]
I hope this feature request does not get lost. Could labels such as `topic/gitea-actions`, `topic/ui`, `topic/ui-interaction` and/or `type/enhancement` be added?

**Comment by silverwind** [comment]
https://github.com/go-gitea/gitea/pull/37744
